src/chromo/common.py

```diff
@@ -36,55 +36,72 @@
 # If we want this, it should be computed dynamically via a property.
 @dataclasses.dataclass
 class CrossSectionData:
-    """Information of cross-sections returned by the generator.
+    """Information of cross sections returned by the generator.
 
     The class is driven by the amount of detail that Pythia-8 provides.
     Most other generators do not fill all attributes. When information is missing,
     the attributes contain NaN. The fields are intentionally redundant, since
-    some generators may provide only the total cross-section or only the inelastic
-    cross-section.
+    some generators may provide only the total cross section or only the inelastic
+    cross section.
 
-    All cross-sections are in millibarn.
+    All cross sections are in millibarn.
 
     Attributes
     ----------
     total : float
-        Total cross-section (elastic + inelastic).
+        Total cross section (elastic + inelastic).
     inelastic : float
-        Inelastic cross-section. Includes diffractive cross-sections.
+        Inelastic cross section. Includes diffractive cross sections.
     elastic : float
-        Cross-section for pure elastic scattering of incoming particle
+        Cross section for pure elastic scattering of incoming particle
         without any new particle generation.
+    quasielastic : float
+        Nuclear cross section defined as total minus quasielastic.
     diffractive_xb : float
-        Single diffractive cross-section. Particle 2 remains intact,
+        Single diffractive cross section. Particle 2 remains intact,
         particles are produced around Particle 1.
     diffractive_ax : float
-        Single diffractive cross-section. Particle 1 remains intact,
+        Single diffractive cross section. Particle 1 remains intact,
         particles are produced around Particle 2.
     diffractive_xx : float
-        Double diffractive cross-section. Central rapidity gap, but
+        Double diffractive cross section. Central rapidity gap, but
         paricles are producted around both beam particles.
     diffractive_axb : float
-        Central diffractive cross-section (e.g.
+        Central diffractive cross section (e.g.
         pomeron-pomeron interaction.)
+    diffractive_sum : float
+        Sum of diffractive cross sections.
     """
 
     total: float = np.nan
     inelastic: float = np.nan
     elastic: float = np.nan
+    prod: float = np.nan
+    quasielastic: float = np.nan
```

Comment:
Could you add the description for `prod`?

Reply:
Added.